internal/client/client.go

```diff
@@ -127,6 +122,18 @@ func (c *CSAPI) SendEventSynced(t *testing.T, roomID string, e b.Event) string {
 	return eventID
 }
 
+// SendEventSynced sends `e` into the room.
+func (c *CSAPI) SendEvent(t *testing.T, roomID string, e b.Event) *http.Response {
```

Comment:
Please move this into a helper function inside `tests/client_power_levels_test.go` and not inside the Complement `CSAPI` client struct.

The reason for this is because `CSAPI` should be reserved for functions which are commonly used across many tests, and this function is primarily only useful to check error conditions when sending events. In addition, by having both forms sit alongside each other in `CSAPI`, it confuses test authors who may prefer to use this function over `SendEventSynced` which would end up being disasterous as `SendEventSynced` is the only form which guards against race conditions when processing new events.